src/VolleyManagement.WebClient/spa/index.html

```diff
@@ -3,19 +3,17 @@
 
 <head>
   <meta charset="utf-8">
+  <script src="https://apis.google.com/js/platform.js"></script>
```

Comment:
What is the purpose of this one?

Reply:
Google Platform Library used to integrate Google Sign-In on web page.